lambdas/appointment_email.py

```diff
@@ -145,12 +163,15 @@ def send_cal_email(supp, supp_email, student, student_email, appt_time, duration
         attendees = []
         attendees.append(requester)
     if toCancel:
-        eml_body = appt_type+" appointment between "+supp+" and student "+student+" from "+ddtstart.strftime("%B %d, %Y from %I:%M %p")+" to "+ddtend.strftime("%I:%M %p")+" has been cancelled."
-    
+        eml_body = appt_type+" appointment between "+supp+" and student "+student+" from "+ddtstart.strftime("%B %d, %Y from %I:%M %p")+" to "+ddtend.strftime("%I:%M %p")+" has been cancelled.                                                   "
+        #eml_body = '<html><body><p> appt_type+" appointment between "+supp+" and student "+student+" from "+ddtstart.strftime("%B %d, %Y from %I:%M %p")+" to "+ddtend.strftime("%I:%M %p")+" has been cancelled."</p>' +'<p>"Comment: "+comme </p>' + '</body></html>'
     else:    
-        eml_body = appt_type+" appointment between "+supp+" and student "+student+" has been scheduled for "+ddtstart.strftime("%B %d, %Y from %I:%M %p")+" to "+ddtend.strftime("%I:%M %p")
+        eml_body = appt_type+" appointment between "+supp+" and student "+student+" has been scheduled for "+ddtstart.strftime("%B %d, %Y from %I:%M %p")+" to "+ddtend.strftime("%I:%M %p") + ".                                                  "
+       # eml_body = <html><body><p> appt_type+" appointment between "+supp+" and student "+student+" from "+ddtstart.strftime("%B %d, %Y from %I:%M %p")+" to "+ddtend.strftime("%I:%M %p")+" has been cancelled."</p> + <p>"Comment: "+ comme </p> + </body></html>
+       
     eml_body_bin = "This is the email body in binary - two steps"
     msg = MIMEMultipart('mixed')
+    
```

Comment:
Why add an extra blank line here? It is breaking the grouping of msg variable assignments.